Src/FluentAssertions/AssertionExtensions.cs

```diff
@@ -67,7 +67,8 @@ public static Func<Task> Awaiting<T>(this T subject, Func<T, Task> action)
             return () => action(subject);
         }
 
-#if NETCOREAPP2_1 || NETSTANDARD2_1
+        #if !NETSTANDARD2_0
```

Comment:
`ValueTask` and `ValueTask<T>` should be available as the netstandard2.0 target has a dependency on `System.Threading.Tasks.Extensions`.